tests/io/vasp/test_help.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import annotations
+
+import io
+from unittest.mock import patch
+
+import pytest
+import requests
+
+from pymatgen.io.vasp.help import VaspDoc
+
+try:
+    from bs4 import BeautifulSoup
```

Comment:
could also use `pytest.importorskip`

Reply:
Sure. Currently our unit tests seem to use multiple mechanism to conditionally skip test if some module is not available. I might keep this into my TODO list to unify this.